src/site/content/en/learn/css/selectors/index.md

```diff
@@ -0,0 +1,486 @@
+Instruct CSS to look for attributes by wrapping the selector with square brackets (`[ ]`).
+
+```css
+[data-type="primary"] {
+	color: red;
+}
+```
+
+This CSS looks for all elements that have an attribute of `data-type` with a value of `primary`, like this:
+
+```html
+<div data-type="primary"></div>
+```
+
+Instead of looking for a specific value of `data-type`,
+you can also look for elements with the attribute present, regardless of its value.
+
+```css
+[data-type] {
+	color: red;
+}
+```
+
+```html
+<div data-type="primary"></div>
+<div data-type="secondary"></div>
+```
+
+Both of these `<div>` elements will have red text.
+
+You can use case-sensitive attribute selectors
+by adding an `s` operator to your attribute selector.
+
+```css
+[data-type="primary" s] {
+	color: red;
+}
+```
+
+This means that if a HTML element had a `data-type` of `Primary`,
+instead of `primary`, it would not get red text.
+You can do the opposite—case insensitivity—by using an `i` operator.
+
+Along with case operators,
+you have access to operators that match portions of strings inside attribute values.
+
+```css
+
+/* A href that contains “example.com” */
+[href*="example.com"] {
+	color: red;
+}
+
+/* A href that starts with https */
+[href^="https"] {
+	color: green;
+}
+
+/* A href that ends with .com */
+[href$=".com"] {
+	color: blue;
+}
+```
+
+<figure class="w-figure">
+  {% Codepen {
+    user: 'web-dot-dev',
+    id: 'BapBbOy'
+  } %}
+  <figcaption class="w-figcaption">In this demo, the `$` operator in our attribute selector gets the filetype from the `href` attribute.
+  This makes it possible to prefix the label—based on that filetype—using a pseudo-element.</figcaption>
+</figure>
+
+### Grouping selectors
+
+A selector doesn't have to match only a single element.
+You can group multiple selectors—separating them with commas—like this:
+
+```css
+strong, em, .my-element, [lang] {
+	color: red;
+}
+```
+
+This example extends the color change to both `<strong>` elements and `<em>` elements.
+It's also extended to a class named `.my-class`,and an element that has a `lang` attribute.
+
+## Pseudo-classes and pseudo-elements
+
+CSS provides useful selector types that focus on specific platform state,
+like when an element is hovered, structures _inside_ an element, or parts of an element.
+
+### Pseudo-classes
+
+HTML elements find themselves in various states,
+either because they are interacted with,
+or one of their child elements is in a certain state.
+
+For example, an HTML element could be hovered with the mouse pointer by a user
+*or* a child element could also be hovered by the user.
+For those situations, use the `:hover` pseudo-class.
+
+```css
+/* Our link is hovered */
+a:hover {
+	outline: 1px dotted green;
+}
+
+/* Sets all even paragraphs to have a different background */
+p:nth-child(even) {
+	background: floralwhite;
+}
+```
+
+We cover pseudo-classes in more detail in a later lesson.
```

Comment:
Let's rephrase to something like 

"For more on pseudo classes, check out lesson 15" <-- with a link to that lesson

Add a @@TODO to link it up